db/schema.rb

```diff
@@ -24,7 +24,7 @@
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
     t.string   "slug"
-    t.datetime "closes_at"
+    t.datetime "closes_at",      null: false
```

Comment:
I'm not really sure why `rake db:migrate` locally generated that. I think there should be `closes_at`, but I'll doublecheck.